src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -3246,6 +3249,24 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Convert_number" xml:space="preserve">
     <value>Convert number</value>
   </data>
+  <data name="Code_analysis_improvements_0" xml:space="preserve">
+    <value>Code analysis improvements ({0})</value>
+  </data>
+  <data name="Do_not_show_Code_analysis_improvements" xml:space="preserve">
+    <value>Do not show 'Code analysis improvements'</value>
+  </data>
+  <data name="Code_analysis_improvements_diagnostic_id_based_title" xml:space="preserve">
```

Comment:
It's unclear what the number is, we probably need to be a bit more explicit on their meaning.

Reply:
I feel given our preferred design in the latest meeting, it is going to be difficult to explain what the count means succinctly.
We decided that we are _not_ going to compute project/solution wide diagnostics for performance reasons. Additionally, we will just rely on the current background analysis option to determine if we are using diagnostics for active document OR all open documents OR entire project.
We want to be explicitly careful to not add any performance overhead here, so the suggestions shown will _not force compute_ any diagnostics but instead work off diagnostics computed from background analysis.

All this means that the count of suggestions shown in this lightbulb item will be variable/stateful for the same project snapshot based on what documents are open, the background analysis option value, and what documents have been analyzed and diagnostics cached by background analysis. I think that is a reasonable experience though and conveys - `we found 'x' suggestions/improvements in one or more documents based on current state of background analysis`.